servers/src/test/java/tachyon/worker/block/allocator/AllocatorContractTest.java

```diff
@@ -38,11 +38,12 @@
 public class AllocatorContractTest extends BaseAllocatorTest {
   protected List<String> mStrategies;
 
-  @Before
   /**
    *  Try to find all implementation classes of {@link Allocator} in the same package
    */
-  public void setup() {
+  @Before @Override
```

Comment:
Before Override should be on their own line according to Google java style guide